perception/occupancy_map_monitor/src/occupancy_map_monitor.cpp

```diff
@@ -303,6 +318,7 @@ bool OccupancyMapMonitor::loadMapCallback(moveit_msgs::LoadMap::Request& request
   tree_->lockWrite();
   try
   {
+    tree_->enableChangeDetection(false);
```

Comment:
I might be missing something, but why isn't this part of #602 then? It seems unrelated to this feature as it goes into the loadMapCallback and that is already present in the original code.

In general - is this even required? The only problem that I see would be that a tree load with enabled change detection does not reflect the full tree change correctly.